Update readme files (#1)
DevExpress-Examples/getting-started-with-scheduler: angular/README.md

## Running unit tests

> Run `ng test` to execute the unit tests using [Karma](https://karma-runner.github.io).

Reviewer: ```suggestion
Run `ng test` to execute unit tests using [Karma](https://karma-runner.github.io).
```

## Running end-to-end tests

> Run `ng e2e` to execute the end-to-end tests using [Protractor](http://www.protractortest.org/).

Reviewer: ```suggestion
Run `ng e2e` to execute end-to-end tests using [Protractor](http://www.protractortest.org/).
```

## Further help

> To get more help on the Angular CLI, use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.

Reviewer: ```suggestion
To get more help on the Angular CLI, use `ng help` or review the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
```
